ci-operator/step-registry/assisted/baremetal/images/publish/assisted-baremetal-images-publish-commands.sh

```diff
@@ -64,16 +66,16 @@ echo "INFO Image tag is $IMAGE_TAG"
 # Setup registry credentials
 REGISTRY_TOKEN_FILE="$SECRETS_PATH/$REGISTRY_SECRET/$REGISTRY_SECRET_FILE"
 
-config_file="$HOME/.docker/config.json"
+config_file="${XDG_RUNTIME_DIR}/containers/auth.json"
```

Comment:
`assisted-baremetal-images-publish`, `assisted-baremetal-operator-publish`, `opendatahub-io-ci-image-mirror` and `red-hat-data-services-ci-image-mirror` seem to be used only up to 4.10 so these should be okay to change